docs/source/architecture/journal.rst

```diff
@@ -2,476 +2,196 @@
 Journal
 *******
 
-The Journal is responsible for maintaining and extending the blockchain for the
-validator. This responsibility involves validating candidate blocks, evaluating
-valid blocks to determine if they are the correct chain head, and generating
-new blocks to extend the chain.
+The `journal` is the group of validator components that work together to handle
+batches and proposed blocks. These components are responsible for completing
+published blocks, publishing batches into blocks to extend the chain, and
+validating proposed blocks to determine if they are the correct chain head.
 
-The Journal is the consumer of Blocks and Batches that arrive at the validator.
-These Blocks and Batches arrive via interconnect, either through the gossip
-protocol or the REST API. The newly-arrived Blocks and Batches are sent to the
-Journal, which routes them internally.
+Blocks and batches arrive via interconnect, either through the gossip protocol
+or the REST API. The processing of these blocks and batches is handled in
```

Comment:
change "or the REST API" to "or from client requests" since clients may submit batches directly to the validator, and the way this is phrased makes it sound like the REST API is required."